cli/ballerina-cli/src/test/resources/test-resources/command-outputs/unix/build-bal-project-with-build-tool-not-found.txt

```diff
@@ -1,6 +1,6 @@
 
 Executing Build Tools
-WARNING: Skipping validation of tool options for tool grpc(generate-grpc-client) due to: Schema file not found: grpc-options-schema.json
-ERROR [grpc] Build tool 'grpc' not found
+ERROR [grpc] Build tool 'grpc' cannot be resolved. Please try again without '--offline' flag
```

Comment:
can this happen when there's no internet connectivity even though `--offline` is not set explicitly?

Reply:
Let's remove the part `Please try again...`.

If the tool is not there in the central, this can happen. Also, we don't print this kind of message for dependency resolution. So it will be consistent to not have it.

Reply:
No, this only happens when the offline flag is provided. Removed the `Please try..` part. So now both messages for unresolved tools offline and online will be same.